content/en/docs/tasks/administer-cluster/namespaces.md

```diff
@@ -111,213 +111,209 @@ More information on `finalizers` can be found in the namespace [design doc](http
 
 1. Delete a namespace with
```

Comment:
Does this need to stay as a numbered list? I think for this particular section I'd remove the list numbering and leave the indentation (down to line 122) as-in.

Reply:
Thanks @sftim  - We do not need a numbered list for this item. Removed as suggested.